events/class_tests/event_tests.py

```diff
@@ -58,3 +58,30 @@ def test_exist_event(self):
     def test_invalidation(self, title, date_time_start, date_time_end, expected_error):
         with pytest.raises(ValidationError, match=expected_error):
             create_event(title, date_time_start, date_time_end).save()
+
+    def test_get_all_meetings(self):
+        assert Event.objects.get(title="event1") in Event.objects.get_all_meetings()
+        assert Event.objects.get(title="event3") in Event.objects.get_all_meetings()
```

Comment:
instead of repeating the code, you can do it like:
```suggestion
        for event_title in ("event1", "event3"):
            assert Event.objects.get(title=event_title) in Event.objects.get_all_meetings()
```

Reply:
done. thanks 😄